docs/openapi/components/schemas/common/EntrySummary.yml

```diff
@@ -0,0 +1,396 @@
+    type: string
+    $linkedData:
+      term: exportingCountry
+      '@id': https://schema.org/addressCountry
+  exportDate:
+    title: Export Date
+    description: For merchandise exported by vessel, record the month, day, and year, on which the carrier departed the last port in the exporting country.
+    type: string
+    $linkedData:
+      term: exportDate
+      '@id': https://schema.org/Date
+  immediateTransportationNumber: 
+    title: Immediate Transportation Number
+    description: Record the IT number obtained from the CBP Form 7512, AWB number from theTransit Air Cargo Manifest (TACM), or Automated Manifest System (AMS) master in-bond (MIB) movement number. 
+    type: string
+    $linkedData:
+      term: immediateTransportationNumber
+      '@id': https://schema.org/identifier
+  immediateTransportationDate:
+    title: Immediate Transportation Date
+    description: Record the month, day, and year, obtained from the CBP Form 7512,TACM, or AMS MIB record.
+    type: string
+    $linkedData:
+      term: immediateTransportationDate
+      '@id': https://schema.org/Date
+  missingDocuments: 
+    title: Missing Documents
+    description: Record the appropriate document code number(s) to indicate documents not availableat the time of filing the entry summary. A maximum of two codes may be used. Thebond charge should be made on the entry summary only for those documents requiredto be filed with the entry summary.
+    type: array
+    items: 
+      type: string 
+    $linkedData:
+      term: missingDocuments
+      '@id': https://w3id.org/traceability#missingDocuments
+  portOfLoading:
+    title: Port Of Loading
+    description: For merchandise arriving in the U.S. by vessel, record the five digit numeric code listedin the “Schedule K” (Classification of Foreign Ports by Geographic Trade Area andCountry) for the foreign port at which the merchandise was actually laden on the vesselthat carried the merchandise to the U.S. The “Schedule K” may be retrieved at:http://www.navigationdatacenter.us/wcsc/scheduleK/schedulek.htmIf the actual port name is not provided, use the code for "all other ports" for the countryin which the merchandise was laden on the vessel that carried the merchandise to the U.S.
+    $ref: ./Place.yml
+    $linkedData:
+      term: portOfLoading
+      '@id': https://schema.org/Place
+  portOfUnlading:
+    title: Port Of Unlading
+    description: Record the U.S. port code where the merchandise was unladen from the importingvessel, aircraft or train. Do not show the name of the port instead of the numeric code.
```

Comment:
```suggestion
    description: Record the U.S. port code where the merchandise was unladen from the importing vessel, aircraft, or train. Do not show the name of the port instead of the numeric code.
```